strongbox-web-core/src/test/resources/temp/metadata-faa-maven-plugin/META-INF/maven/plugin.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><plugin>
```

Comment:
What are these two XML-s for here? Are they related to the PR?

Reply:
It's some test rubbish - haven't noticed it, sorry :) Removed.